charts/tempo-distributed/templates/enterprise-gateway/_helpers.tpl

```diff
@@ -0,0 +1,33 @@
+{{/*
+Return the appropriate apiVersion for ingress.
+*/}}
+{{- define "tempo.ingress.apiVersion" -}}
```

Comment:
FYI We've in Mimir recently made the breaking change to only support kube >= 1.20. Since we are not actually testing on older versions - so we could get rid of this check altogether.

Reply:
This sounds good to me.  Perhaps a follow up breaking change to upgrade the required version for the chart as well as drop the api version comparisons.